### ISSUE TITLE
"Loading" indicator while variables are being fetched

### ISSUE THREAD

**Issue by RobinDaugherty** [charliesome/better_errors#382]
When the console is first opened, or once the user has selected a new frame, the variable-fetching request can take a long time. Currently, there is no indication that the request is in progress.

**Comment by bquorning** [comment]
> the variable-fetching request can take a long time.

Please see #360 for a proposed fix for this performance issue.

**Comment by RobinDaugherty** [comment]
Good point @bquorning, I should have mentioned that. My hope is to solve the UI on top of that, so that on slow development machines for example, there's still some indication that the request is in progress. I recently solved a lack of feedback when the variable request fails under certain conditions (#389), which is a good start, but it could still be improved.

**Comment by bquorning** [comment]
That sounds good indeed.

And can I just say this: I’m so happy to see this project being actively maintained again ❤️
